examples/managementcli/main.go

```diff
@@ -57,10 +58,14 @@ func userCreate(args []string) error {
 		tenants = append(tenants, &descope.AssociatedTenant{TenantID: tenantID})
 	}
 	user := &descope.UserRequest{}
-	user.Email = "foo@bar.com"
+	user.Email = flags.Email
 	user.Phone = flags.Phone
+	if flags.Email == "" && flags.Phone == "" {
+		user.Email = "foo@bar.com" // email or phone are required
```

Comment:
Why are we not just throwing an error?

Reply:
thought about this as well, but it makes the API of the cli a bit more complex, I believe that this is the reason we add the default email in the first place